src/odbc/src/app/application_data_buffer.cpp

```diff
@@ -310,6 +310,20 @@ ConversionResult::Type ApplicationDataBuffer::PutStrToStrBuffer(
     assert(false);
   }
 
+  if (isTruncated) {
+      if (resLenPtr) {
+        *resLenPtr = SQL_NO_TOTAL;
```

Comment:
While this is correct, it's actually more informative to the app to report the exact remaining bytes (which I believe we calculated earlier in the function). That lets the app create a buffer of the appropriate size instead of creating an unknown-sized buffer.

Reply:
I have updated to not use SQL_NO_TOTAL!